test/src/loaders/workspace_uploader/workspace_uploader_test.py

```diff
@@ -356,25 +414,6 @@ def test_upload_assembly_files_in_parallel(setup_and_teardown):
         }
     )
 
-    # check softlink for post_process
-    assert os.readlink(os.path.join(upload_dir, "12345_58_1")) == os.path.join(
-        output_dir, "12345_58_1"
-    )
-    assert os.readlink(os.path.join(upload_dir, "12345_60_1")) == os.path.join(
-        output_dir, "12345_60_1"
-    )
-
-    # check hardlink for post_process
-    assert os.path.samefile(
-        src_files[0],
-        os.path.join(output_dir, "12345_58_1", "12345_58_1.fna.gz")
-    )
-
-    assert os.path.samefile(
-        src_files[1],
-        os.path.join(output_dir, "12345_60_1", "12345_60_1.fna.gz")
-    )
-
```

Comment:
This is deleted because if only assemblies are uploaded nothing happens in the WS source dir now? Presumably when we upload genomes there will be a new test along these lines?

Reply:
yup. that's my plan.

Reply:
:+1: